.github/scripts/pr_body_check.py

```
@@ -0,0 +1,81 @@
+    return incomplete_sections
+
+def main() -> None:
+    pr_body = os.environ.get('PR_BODY', '')
+    if not pr_body:
+        warn('PR description is missing! Please provide a valid PR description.')
+        return
+
+    template_path = os.path.join(
+        os.environ.get('GITHUB_WORKSPACE', '.'),
+        '.github',
+        'pull_request_template.md'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)